sdk/storage/azure-storage-blob-cryptography/src/test/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlockBlobAsyncApiTests.java

```diff
@@ -153,8 +153,8 @@ public void v2DownloadTest(int dataSize) {
 
 
         StepVerifier.create(bec.uploadWithResponse(new BlobParallelUploadOptions(Flux.just(data.duplicate())))
-                .then(FluxUtil.collectBytesInByteBufferStream(bec.downloadStream())))
-            .assertNext(r -> assertArraysEqual(data.array(), r))
+                .then(bec.downloadContentWithResponse(null, null)))
```

Comment:
For both the `downloadContentWithResponse` and `downloadContent` methods, it would probably be worth just adding new, standalone test cases for them instead of switching the methods used for these existing test cases. Mainly, by having the test standalone, we reduce the chance of lowering coverage for the other `downloadStream` method and it does a better job of preserving coverage in the future (e.g. in the case, later down the road we change the method use in these and unintentionally remove coverage for the download content methods).

In terms of how to introduce these new test cases, I don't have a preference on whether we copy these tests or even write a simpler version that just uploads and downloads a blob with the target download method.